### ISSUE TITLE
Horizontal screen problem

### ISSUE THREAD

**Issue by GrapeFruitJun** [daved01/Driver-Assist#18]
When my phone is in landscape, the ratio of CAShapelayer is not correct, how to correct it, I hope to get your answer, thank you!

**Comment by daved01** [comment]
Thank you for your comment! I will take a look within the next few days.

**Comment by GrapeFruitJun** [comment]
> 

Thank you for your reply and I look forward to your answer!

**Comment by GrapeFruitJun** [comment]
I'm sorry to bother you, I made some attempts myself and still didn't get the right result, still hoping to get your feedback

**Comment by daved01** [comment]
I just checked and remembered that I fixed the orientation in the info.plist to portrait because I did not deal with resizing and getting the device orientation. So it seems to me the steps would be:
- Allow landscape view
- Get device orientations
- Rotate the video view and shapelayer based on current orientation
- Rescale the boxes based on orientation
Feel free to implement this and submit a PR. Sorry if I cannot help more at the moment. When I find the time I will take a look myself.
